content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -20,257 +17,246 @@ typically requires a VPC or a VPN, as well as a container network that
 provides direct (without NAT or firewall deny) routing to the endpoints. The machine
 is not required to have access to the cluster IP addresses assigned by Kubernetes.
 
-* The Istio control plane services (Pilot, Mixer, Citadel) and Kubernetes DNS server must be accessible
-from the VMs. This is typically done using an [Internal Load
-Balancer](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer).
-You can also use NodePort, run Istio components on VMs, or use custom network configurations,
-separate documents will cover these advanced configurations.
+* The VM must have access to a DNS server that resolves names to cluster IP addresses. Options
+include exposing the Kubernetes DNS server through an internal load balancer, using a Core DNS
+server, or configuring the IPs in any other DNS server accessible from the VM.
 
 ## Installation steps
 
 Setup consists of preparing the mesh for expansion and installing and configuring each VM.
+We'll use /etc/hosts as an example in this guide, it is the easiest way to get things working.
+
+### Preparing the Kubernetes cluster for expansion
 
-An example script to help with Kubernetes setup is available as part of the release bundle called
-[`install/tools/setupMeshEx.sh`]({{< github_file >}}/install/tools/setupMeshEx.sh). Check the script content and environment variables supported (like GCP_OPTS).
+The following commands must be run on a machine with admin privileges to the cluster.
 
-An example script to help configure a machine is available as part of the release bundle called [`install/tools/setupIstioVM.sh`]({{< github_file >}}/install/tools/setupIstioVM.sh).
-You should customize it based on your provisioning tools and DNS requirements.
+*  If `--set global.meshExpansion=true` was not specified at install, re-apply the
+   template or 'helm upgrade' with the option enabled.
 
-### Preparing the Kubernetes cluster for expansion
+    {{< text bash >}}
+
+    $ cd install/kubernetes/helm/istio
+    $ helm upgrade --set global.meshExpansion=true --values myvalues.yaml istio-system .
```

Comment:
myvalues.yaml doesn't exist - there's a wide variety of other values files, which ones should I use?

Reply:
I am trying it with values.yaml. EXCITING. Now for it to stop saying "Happy Helming!" at me.